app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/dashboard/DashboardScreen.kt

```diff
@@ -102,14 +106,15 @@ fun DashboardScreen(
                     subtitle = "${stringResource(R.string.good)} ${getTimeOfDayGreeting()}",
                     todayInterviewList = todayInterviewList,
                     username = getNameInitials(dashboardViewModel.username),
-                    isInterviewDetailsVisible = isScrollingUp,
-                    navController = navController,
-                    onAvatarClick = {
-                        navController.navigate(AppScreens.MainScreens.Profile.route) {
-                            popUpTo(AppScreens.MainScreens.Dashboard.route)
-                        }
+                    isInterviewDetailsVisible = derivedStateOf {
+                        isScrollingUp.value || selectedInterviewList.value.size <= 3
```

Comment:
We would want to avoid hardcoding the value of 3 , otherwise depending on phone size this problem might happen again. For now I am approving this and merging it but think of a generic solution to find out this number to compare with selectedInterviewList.